src/ui/components/Library/Team/View/ViewPage.tsx

```diff
@@ -19,19 +16,14 @@ export function ViewPage({ defaultView }: { defaultView: string }) {
 
 export function ViewPageContent() {
   const { view } = useContext(ViewContext);
-  const [enableTestSuitesNewRunsView] = useLocalStorageUserData("enableTestSuitesNewRunsView");
   return (
     <div className="flex flex-grow flex-col overflow-hidden">
       <FilterBarContainer />
       <div className="flex flex-grow flex-row overflow-hidden">
         {view === "recordings" ? (
           <RecordingsPage />
         ) : view === "runs" ? (
-          enableTestSuitesNewRunsView ? (
-            <NewTestRunsPage />
-          ) : (
-            <TestRunsPage />
-          )
+          <TestRunsPage />
```

Comment:
This should be `<NewTestRunsPage />`, yes?

Reply:
It was just aliased to `New` before, I just removed it. https://github.com/replayio/devtools/commit/85db88f03d0c3ad51c3705b77e30ff56d92edfde